common/scala/src/main/scala/whisk/core/database/CouchDbRestStore.scala

```diff
@@ -169,6 +168,8 @@ class CouchDbRestStore[DocumentAbstraction <: DocumentSerializer](
                     transid.finished(this, start, s"[GET] '$dbName' failed to get document: '${doc}'; http status: '${code}'")
                     throw new Exception("Unexpected http response code: " + code)
             }
+        } recoverWith {
```

Comment:
That's for _all_ other errors? Didn't want to match on the applicable exceptions and throw a generic "unknown internal server error" otherwise?